journal/updated/09-chapter-8.adoc

```diff
@@ -299,58 +304,58 @@ Several strangers were there, and I had satisfaction in being with them,
 
 Fourth month 29th, 1793.
 Attended our Quarterly Meeting at Nottingham,
-and 1st of fifth month our Monthly Meeting at Loughborough;
-this little journey I was enabled to perform beyond expectation,
+and 1st of fifth month our Monthly Meeting at Loughborough.
+This little journey I was enabled to perform beyond expectation,
 and felt my mind under the covering of the Spirit of the Lord, and returned in peace.
-My complaints too not feeling so grievous as in months past,
+My complaints not feeling so grievous as in months past,
 I apprehended it my duty to attend some Quarterly Meetings,
-although being still in a very weak state every way, the trial was great.
+but being still in a very weak state, the trial was great.
 On the 15th of sixth month, I left home, accompanied by my son Joseph,
-reached Coventry that evening, sat three meetings on first-day,
+and reached Coventry that evening. Attended three meetings on first-day,
 and the Quarterly Meeting on second.
 
 Went to Warwick on third-day, was at their week-day meeting on fourth,
-and at Eddington meeting on sixth-day, at Shipston meeting on first-day,
-and to Banbury that evening; was at three meetings on secondday,
-and the Quarterly Meeting on third; went same day to Buckingham,
+and at Eddington meeting on sixth-day. At Shipston meeting on first-day,
+and to Banbury that evening; was at three meetings on second-day,
+and the Quarterly Meeting on third. I went the same day to Buckingham,
 was at meeting in the evening, and at the Quarterly Meeting on fourth-day,
 which was large and satisfactory.
-From there to Northampton,
-on fifth-day was at meeting in the evening and at the Quarterly Meeting on sixth-day;
+From there to Northampton;
+on fifth-day was at meeting in the evening, and at the Quarterly Meeting on sixth-day;
 from there to Wellington that evening,
-and lodged at our very kind and honest friend Benjamin Middleton`'s;
-was at Finedon on first-day,
-at their Monthly Meeting on second and on third-day to Ridlington Park;
-from there to Oakham, at their meeting on fifth-day, and on sixth,
+and lodged at our very kind and honest friend Benjamin Middleton`'s.
+Was at Finedon on first-day,
+at their Monthly Meeting on second, and on third-day to Ridlington Park.
+From there to Oakham, at their meeting on fifth-day, and on sixth,
 after having a solemn opportunity with our dear friends there, returned to the Park,
-and on seventh-day, in a renewed sense of Divine love we took leave.
+and on seventh-day, in a renewed sense of Divine love, we took leave.
 On first-day was at their meetings at Leicester; from there to Groby Lodge;
 on fourth-day returned to Leicester,
-it having rested on my mind to be there the first-day following:
-was taken very unwell at meeting; lodged at John Priestman`'s,
-whose wife took great care of me; and getting better,
+it having rested on my mind to be there the first-day following.
+I became very ill at meeting, and lodged at John Priestman`'s,
```

Comment:
This is fine, but "I was taken very unwell" is also a common thing in these journals, and you can probably leave it in the future.